03-programming.Rmd

```diff
@@ -470,7 +470,7 @@ Figure \@ref(fig:3-2) highlights that voting patterns are clearly not uniform be
 
 ### Type consistency
 
-When programming, it is helpful if the return value from a function always takes the same form. Unfortunately, not all base R functions follow this idiom. For example, the functions `sapply()` and `[.data.frame()` aren't type consistent
+When programming, it is helpful if the return value from a function always takes the same form. Unfortunately, not all base R functions follow this idiom. For example, the functions `sapply()` and `[.data.frame()` aren't type consistent.
```

Comment:
I think these are deliberate because the code follows-on from the comment. Suggestion: replace your new periods `.` with colons `:`.